### PR TITLE
GEOPY-1165: b

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-1165 - Check for properties for Property Group Type - Implement Property Group Type: Simple**
Reverting to Enum

### CODE REVIEW

File: geoh5py/groups/property_group_type.py [hunk 1]
```diff
@@ -20,13 +20,14 @@
...
 from abc import ABC
+from enum import Enum
 
 from ..data import Data, DataAssociationEnum, FloatData, NumericData
 
 
 class PropertyGroupType(ABC):
     """
-    Class to define the basic structure of a property group type.
+    Abstract base class for defining the type of group that a property group is.
```
Comment (MatthieuCMira) [inline geoh5py/groups/property_group_type.py:30]:
```suggestion
    Class to define the basic structure of a property group type.
```

File: geoh5py/groups/property_group_type.py [hunk 2]
```diff
@@ -132,26 +133,48 @@ def verify(cls, children: list[Data]):
             )
 
 
-# mapper for group types
 GROUP_TYPES = {
```
Comment (MatthieuCMira) [inline geoh5py/groups/property_group_type.py:136]: I have the feeling there is some repetition here betwen GROUP_TYPE, the objects of the Enum and the name

**Review by MatthieuCMira — DISMISSED** [review]
As I did it, I am not against it. 
Just feel there is a bit of repetition in property_group_type.

Do you have an idea to make it simpler?

File: geoh5py/groups/property_group_type.py (hunk 2 above)
Comment (domfournier) [reply geoh5py/groups/property_group_type.py:136]: Right, we can probably remove the name from the classes....

**Comment by MatthieuCMira** [comment]
I'm good with the changes

**Review by MatthieuCMira — APPROVED** [review]

**Pull request merged**